scp/scp.go

```diff
@@ -0,0 +1,102 @@
+package scp
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/ONSdigital/dp-cli/aws"
+	"github.com/ONSdigital/dp-cli/config"
+	"github.com/ONSdigital/dp-cli/out"
+)
+
+// Options holds the state of flags given
+type Options struct {
+	IsPull      *bool
+	IsRecursing *bool
+	IsConfirmed *bool
+	Verbosity   *int
+}
+
+func withCWD(file string) (string, error) {
+	if file[0] == '/' {
+		return file, nil
+	}
+	var pwd string
+	var err error
+	if pwd, err = os.Getwd(); err != nil {
+		return "", err
+	}
+	return filepath.Join(pwd, file), nil
```

Comment:
As below, `pwd` is a command. 
```suggestion
	var cwd string
	var err error
	if cwd, err = os.Getwd(); err != nil {
		return "", err
	}
	return filepath.Join(cwd, file), nil
```